adk/ADK.py

```diff
@@ -64,20 +65,17 @@ def wrap_binary_data(self, data):
 
     def format_response(self, response):
         if self.is_binary(response):
-            content_type = 'binary'
-            response = base64.b64encode(response)
-            if not isinstance(response, six.string_types):
-                response = str(response, 'utf-8')
-        elif isinstance(response, six.string_types) or isinstance(response, six.text_type):
-            content_type = 'text'
+            content_type = "binary"
+            response = str(base64.b64encode(response), "utf-8")
+        elif isinstance(response, six.string_types) or isinstance(
+            response, six.text_type
```

Comment:
This one specifically, I agree with your changes but this is definitely much harder to read; I'd agree on potentially a new line after the `or` but multi-line separation of a single command is tough to read

Reply:
Here I meant we don't need to do 
```
if not isinstance(response, six.string_types):
    response = str(response, "utf-8")
```
because this `isinstance` check would never be true, as response is a bytes object at this point in code. So the intention was to remove the call to `isinstance`. If you want to multiline you would say:
```
response = base64.b64encode(response)
response = str(response, "utf-8")
```

Reply:
The answer to this is at the bottom of the PR for some reason but let me paste here too:
Here I meant we don't need to do

```
if not isinstance(response, six.string_types):
    response = str(response, "utf-8")

```because this `isinstance` check would never be true, as response is a bytes object at this point in code. So the intention was to remove the call to `isinstance`. If you want to multiline you would say:

```
response = base64.b64encode(response)
response = str(response, "utf-8")
```